Scripts/Api.cs

```diff
@@ -223,5 +416,51 @@ public CreateConversationResponse(string conversationUuid)
         {
             this.ConversationUuid = conversationUuid;
         }
+    }
+
+    public class GetMessageHistoryResponse
+    {
+        public MessageEvent[] Messages { get; }
+
+        [JsonConstructor]
+        public GetMessageHistoryResponse(MessageEvent[] messages) 
+        {
+            Messages = messages;
+        }
+    }
+
+    public class ForkConversationResponse
+    {
+        /// <summary>
+        /// The new forked playthrough token.
+        /// </summary>
+        public string NewToken { get; }
+
+        /// <summary>
+        /// The UUID of the playthrough.
+        /// </summary>
+        public string PlaythroughUuid { get; }
+
+        [JsonConstructor]
+        public ForkConversationResponse(string token, string playthroughUuid)
+        {
+            NewToken = token;
+            PlaythroughUuid = playthroughUuid;
+        }
+    }
+
+    public class GetPlaythroughInfoResponse
+    {
+        public Emotion[] Emotions { get; }
+        public Memory[] Memories { get; }
+        public Impact[] Impacts { get; }

```

Comment:
We can remove `Impact[]` and any reference to impact as these are in app stories only